galleries/examples/images_contours_and_fields/colormap_normalizations.py

```diff
@@ -13,85 +13,86 @@
 
 import matplotlib.colors as colors
 
-# %%
-# Lognorm: Instead of pcolor log10(Z1) you can have colorbars that have
-# the exponential labels using a norm.
-
 N = 100
-X, Y = np.mgrid[-3:3:complex(0, N), -2:2:complex(0, N)]
 
-# A low hump with a spike coming out of the top.  Needs to have
-# z/colour axis on a log scale, so we see both hump and spike.
-# A linear scale only shows the spike.
+# %%
+# LogNorm
+# -------
+# This example data has a low hump with a spike coming out of its center. To see both
```

Comment:
I suggest to always use the pattern: data -> what is the issue with linear norm -> which norm to use instead. I.e.

> This example data has a low hump with a spike coming out of its center. using a linear color scale only shows the spike. To see both hump and spike, this requires the z/colour axis on a log scale. Instead of transforming the data with ``pcolor(log10(Z))``, the color mapping can be logarithmic by using a `.LogNorm`.

And then first plot the linear scale and then the log scale (also for all subsequent examples.

Reply:
Oh yeah, I agree. I found it confusing that the "good" plot comes before the "bad" one. Actually, side-by-side plots (linear on the left, fancy on the right) might be best.